tests/arm/checks/resource/test_FunctionAppsEnableAuthentication.py

```diff
@@ -20,17 +20,18 @@ def test_summary(self):
 
         passing_resources = {
             "Microsoft.Web/sites/config.pass",
+            "Microsoft.Web/sites/config.authsettingsV2"
         }
         failing_resources = {
-            "Microsoft.Web/sites/config.fail",
-            "Microsoft.Web/sites/config.fail2",
+            "Microsoft.Web/sites/config.authsettingsV2",
+            "Microsoft.Web/sites/config.authsettingsV2"
```

Comment:
I don't think that makes sense, the same resource is passing and failing (twice)?

Reply:
resolved offline 👍